libs/feature/editor/src/lib/+state/editor.facade.ts

```diff
@@ -63,4 +66,8 @@ export class EditorFacade {
   setFieldVisibility(field: EditorFieldIdentification, visible: boolean) {
     this.store.dispatch(EditorActions.setFieldVisibility({ field, visible }))
   }
+
+  hasRecordChangedSinceDraft(record: CatalogRecord) {
+    this.store.dispatch(EditorActions.hasRecordChangedSinceDraft({ record }))
+  }
```

Comment:
I'm wondering if the naming could be clearer here between the observable above and the action, as the observable basically expresses the change resulting from the action. Would `checkHasRecordChanged` make sense, for example?